server/game/core/Player.js

```diff
@@ -674,69 +657,20 @@ class Player extends GameObject {
             adjuster.canAdjust(playingType, card, target, ignoreType, penaltyAspect)
         );
         var costIncreases = matchingAdjusters
-            .filter((a) => a.getAmount(card, this) < 0)
-            .reduce((cost, adjuster) => cost - adjuster.getAmount(card, this), 0);
+            .filter((adjuster) => adjuster.direction === CostAdjustDirection.Increase)
```

Comment:
What's the advantage to using an enum to track adjustment direction rather than checking positive/negative?

Reply:
Good callout, looking at it now I think I'm over-engineering and we can just let amount be positive or negative.